src/reservation/azext_reservation/aaz/latest/reservations/reservation_order/_purchase.py

```diff
@@ -0,0 +1,791 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "reservations reservation-order purchase",
```

Comment:
Why use `purchase` instead of `create` for command name?

Reply:
Current `az reservations` is using [`purchase`](https://learn.microsoft.com/en-us/cli/azure/reservations/reservation-order?view=azure-cli-latest#az-reservations-reservation-order-purchase), so keeping it the same here for consistency.